public/js/agora-communication-client.js

```diff
@@ -95,8 +101,7 @@ async function isVideoAvailable() {
   return devices.some(device => 'videoinput' === device.kind);
 }
 
-// video streams for channel
-function createCameraStream(uid, next) {
+function createTmpCameraStream(uid, next){
```

Comment:
Why is this method created twice? the `communication-client.js` and `broadcast-client.js` should only contain unique methods. Reading through the code this looks identical to the `broadcast-client.js`. Common  methods should go into `wp-agora-io-public.js`

Reply:
> Why is this method created twice? the `communication-client.js` and `broadcast-client.js` should only contain unique methods. Reading through the code this looks identical to the `broadcast-client.js`. Common methods should go into `wp-agora-io-public.js`

Fixed. Kindly check it.